src/runtime/google-analytics-event-listener.js

```diff
@@ -73,13 +73,13 @@ export class GoogleAnalyticsEventListener {
       eventLabel: analyticsParams.event_label || gaEvent.eventLabel,
     };
     // TODO(b/234825847): Remove it once universal analytics is deprecated in 2023.
-    const ga = this.win_.ga || null;
-    if (isFunction(ga)) {
+    if (this.constructor.isGaEligible(this.deps_)) {
+      const ga = this.deps_.win().ga || null;
       ga('send', 'event', gaEvent);
     }
 
-    const gtag = this.win_.gtag || null;
-    if (isFunction(gtag)) {
+    if (this.constructor.isGtagEligible(this.deps_)) {
+      const gtag = this.deps_.win().gtag || null;
```

Comment:
if `gtag` is `null`, then line 89 would throw an error, right?

Reply:
Can consolidate into above thread, marking resolved.